src/Components/CardList.tsx

```diff
@@ -297,10 +298,10 @@ export const CardList = ({
             totalPages={totalPages}
             isFirstPage={isFirstPage}
             isLastPage={isLastPage}
-            showFilterBtn
```

Comment:
If we never want to pass this prop into CardListNavigation, we should clean up and remove what it controls in that component.

Reply:
Done :D